### PR TITLE
Some standard superion scrapers

### PR DESCRIPTION
**Pull request by ohinds**

### CODE REVIEW

File: recidiviz/ingest/scrape/regions/us_ga_columbia/us_ga_columbia_scraper.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+                                                     ingest_info)
+
+        for charge in ingest_info.get_all_charges():
+            if charge.status and 'BOND REVOKED' in charge.status:
+                if not charge.bond:
+                    charge.create_bond()
+
+                charge.status = ChargeStatus.PRETRIAL.value
+
+                # Guaranteed to be true, but mypy is too dumb to know that.
```
[the remark this answers is not in the document]
Comment (ohinds) [reply recidiviz/ingest/scrape/regions/us_ga_columbia/us_ga_columbia_scraper.py]: Nice!

File: recidiviz/ingest/scrape/regions/us_ga_coweta/us_ga_coweta_scraper.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+class UsGaCowetaScraper(SuperionScraper):
+    """Scraper implementation for us_ga_coweta."""
+    def __init__(self):
+        super(UsGaCowetaScraper, self).__init__('us_ga_coweta')
+
+    def get_enum_overrides(self):
+        overrides_builder = super(
+            UsGaCowetaScraper, self).get_enum_overrides().to_builder()
+
+        overrides_builder.add('WHITE OR HISPANIC', Race.WHITE)
```
[the remark this answers is not in the document]
Comment (ohinds) [reply recidiviz/ingest/scrape/regions/us_ga_coweta/us_ga_coweta_scraper.py]: ok

File: recidiviz/ingest/scrape/regions/us_ga_dougherty/us_ga_dougherty_scraper.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+            UsGaDoughertyScraper, self).populate_data(content, task,
+                                                      ingest_info)
+
+        person = scraper_utils.one('person', scraped_data.ingest_info) \
+            if scraped_data else Person()
+
+        for booking in person.bookings:
+            for charge in booking.charges:
+                if charge.bond and charge.bond.bond_type:
+                    if not BondType.can_parse(charge.bond.bond_type,
```
[the remark this answers is not in the document]
Comment (ohinds) [reply recidiviz/ingest/scrape/regions/us_ga_dougherty/us_ga_dougherty_scraper.py]: Sure! That was fun.

File: recidiviz/ingest/scrape/regions/us_al_morgan/us_al_morgan_scraper.py [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+        for booking in person.bookings:
+            for charge in booking.charges:
+                if not charge.bond:
+                    continue
+                # TODO (#816) use enum overrides to set the status without code
+                if charge.bond.bond_type and \
+                   charge.bond.bond_type.strip() == 'BOND REVOKED':
+                    charge.bond.status = charge.bond.bond_type
+                    charge.bond.bond_type = None
```
Comment (jamwalla) [inline recidiviz/ingest/scrape/regions/us_al_morgan/us_al_morgan_scraper.py]: will the bond will always be empty at this point? What do you think about clearing the bond or calling `charge.prune()` here?

File: recidiviz/ingest/scrape/regions/us_al_morgan/us_al_morgan_scraper.py [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+
+    def populate_data(self, content, task: Task,
+                      ingest_info: IngestInfo) -> Optional[ScrapedData]:
+        scraped_data: Optional[ScrapedData] = super(
+            UsAlMorganScraper, self).populate_data(content, task, ingest_info)
+
+        person = scraper_utils.one('person', scraped_data.ingest_info) \
+            if scraped_data else Person()
+
+        for booking in person.bookings:
```
Comment (jamwalla) [inline recidiviz/ingest/scrape/regions/us_al_morgan/us_al_morgan_scraper.py]: ooh, also, `ingest_info.get_all_charges(lambda c: c.bond)` if you wanna be fancy

Comment (jamwalla) [reply recidiviz/ingest/scrape/regions/us_al_morgan/us_al_morgan_scraper.py]: though that causes that mypy issue so maybe never mind

File: recidiviz/tools/scraper_template/superion/region_scraper.txt [hunk 6]
```diff
@@ -0,0 +1,26 @@
+# Recidiviz - a platform for tracking granular recidivism metrics in real time
```
[the remark this answers is not in the document]
Comment (ohinds) [reply recidiviz/tools/scraper_template/superion/region_scraper.txt]: Done

File: recidiviz/ingest/scrape/regions/us_ga_forsyth/us_ga_forsyth_scraper.py [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+    """Scraper implementation for us_ga_forsyth."""
+    def __init__(self):
+        super(UsGaForsythScraper, self).__init__('us_ga_forsyth')
+
+    def get_enum_overrides(self):
+        overrides_builder = super(
+            UsGaForsythScraper, self).get_enum_overrides().to_builder()
+
+        overrides_builder.add('ANY BOND', BondType.SECURED)
+        overrides_builder.ignore('FEES (SHERIFF COLLECTED)', BondType)
```
[the remark this answers is not in the document]
Comment (ohinds) [reply recidiviz/ingest/scrape/regions/us_ga_forsyth/us_ga_forsyth_scraper.py]: Done

**Pull request merged**